### ISSUE TITLE
Keyboard navigation doesn't work when filtering

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gavioto/stexbar#198]
```
What steps will reproduce the problem?
1. Suppose I type something using Ctrl+K into the filter
2. Then while focus is still on the filter, I do Alt+Up to go Up, Alt+Left 
arrow key to go back or Alt+Right.
3. In that case, it should go up one level or back. I have to clear the 
contents of the filter box first, or move the focus to the Explorer items view 
to use these navigation keys otherwise they don't work

What is the expected output? What do you see instead?
The expected behavior is that Alt+Up, Alt+Left etc should work regardless of 
the contents of the filter and whether it is filtering or not

What version of the product are you using? On what operating system?
1.8.3.352 on Windows 7 x64

Please provide any additional information below.
Can you please make the Alt+Up etc keyboard shortcuts work even if the filter 
has focus?
```

Original issue reported on code.google.com by `xpclient...@gmail.com` on 22 Jun 2012 at 5:41
